src/ontology/mondo-ingest.Makefile

```diff
@@ -177,6 +171,22 @@ metadata/mondo.sssom.config.yml:
 
 mappings: sssom $(ALL_MAPPINGS)
 
+.PHONY: unmapped-terms-docs
+unmapped-terms-docs: $(foreach n,$(ALL_COMPONENT_IDS), reports/$(n)_unmapped_terms.tsv)
+	python3 $(SCRIPTSDIR)/unmapped_docs.py
+
+.PHONY: unmapped-terms-tables
+unmapped-terms-tables: $(foreach n,$(ALL_COMPONENT_IDS), reports/$(n)_unmapped_terms.tsv)
+
+$(REPORTDIR)/%_unmapped_terms.tsv: $(REPORTDIR)/%_term_exclusions.txt $(REPORTDIR)/mirror_signature-%.tsv $(TMPDIR)/mondo.sssom.tsv metadata/%.yml $(COMPONENTSDIR)/%.owl
```

Comment:
@matentzn What do you think about this design for makefile goals for this feature? Is it OK?
1. `$(REPORTDIR)/%_unmapped_terms.tsv`: Create a table of unmapped terms for each ontology.
2. `unmapped-terms-tables`: Phony target to run all the tables.
3. `unmapped-terms-docs`: Phony target that uses those tables as a prerequisite, calculates statistics, and updates the documentation.

Reply:
I shared w/ Nico on a call and it seems like this design is good.